docs/rules/0192/no-html.md

```diff
@@ -0,0 +1,75 @@
+---
+rule:
+  aip: 192
+  name: [core, '0192', no-html]
+  summary: Comments must not use raw HTML
+permalink: /192/no-html
+redirect_from:
+  - /0192/no-html
+---
+
+# No HTML in comments
+
+This rule enforces that every descriptor in every proto file does not use raw
+HTML in comments, as mandated in [AIP-192][].
+
+## Details
+
+This rule looks at each descriptor in each proto file (exempting oneofs and the
+file itself) and tries to pick up "HTML smell", and complains if it thinks it
```

Comment:
Consider to rephrase it to: "tries to find HTML tags, and complains if it found some." Basically, we need to explain what is "HTML smell".

Reply:
Done.